client/core/trade.go

```diff
@@ -600,12 +628,22 @@ func (t *trackedTrade) isRefundable(match *matchTracker) bool {
 	}
 
 	// Issue a refund if our swap's locktime has expired.
-	swapLocktimeExpired, err := wallet.LocktimeExpired(proof.Script)
+	swapLocktimeExpired, contractExpiry, err := wallet.LocktimeExpired(proof.Script)
 	if err != nil {
 		log.Errorf("error checking if locktime has expired for %s contract on order %s, match %s: %v",
 			dbMatch.Side, t.ID(), match.id, err)
 		return false
 	}
+	if !swapLocktimeExpired {
+		swapCoinID := proof.TakerSwap
+		if dbMatch.Side == order.Maker {
+			swapCoinID = proof.MakerSwap
+		}
+		from := t.wallets.fromAsset
+		log.Infof("Contract for match %v with swap coin %v (%s) has an expiry time of %v (%v), not yet expired.",
```

Comment:
e.g.


> [INF] CORE: Contract for match 32dfe1736e516f62605d81e1be93fc46dc2f08d773f3b0ed54fe63054c99b04c with swap coin 6c2184e067b818e83f93754909b7235c8af69ebc58eb945f3fba90e17dd66ffd:0 (dcr) has an expiry time of 2020-07-31 14:14:10 +0000 UTC (21h23m2s), not yet expired.

Reply:
DCR testnet refund succeeded: https://testnet.dcrdata.org/tx/8bd2219962f983cc621df02c06d95b741e2923512b39bafd51c562f2357e4a97/in/0

Both BTC and DCR testnet refunds have been executed now.